apps/shinkai-desktop/src/store/settings.ts

```diff
@@ -227,6 +229,11 @@ export const useSettings = create<SettingsStore>()(
           set({ playgroundCodePanelSize: size });
         },
 
+        maxChatIterations: 10,
+        setMaxChatIterations: (maxChatIterations) => {
+          set({ maxChatIterations });
+        },
+
```

Comment:
This looks like server state rather than client state, do you think we could manage it with tanstack-query instead?

Reply:
Done

Reply:
could we clean up and remove this state? we no longer need it